scripts/recan_bundle_tdr.py

```diff
@@ -329,18 +353,35 @@ def main(argv):
     Load a canned bundle from DCP/1 and write *.manifest.tdr and *.metadata.tdr
     files showing the desired output for DCP/2.
     """
+    default_version = tdr.Plugin.format_version(datetime(
```

Comment:
shouldn't this be aligned indent? Try extracting a variable to eliminate the wrapping altogether